plan/planbuilder.go

```diff
@@ -923,6 +926,9 @@ func (b *planBuilder) buildLoadData(ld *ast.LoadDataStmt) Plan {
 }
 
 func (b *planBuilder) buildDDL(node ast.DDLNode) Plan {
+	p := &DDL{Statement: node}
```

Comment:
Why move this here?

Reply:
Because I need to use in case *ast.CreateViewStmt